### ISSUE TITLE
Turn off Enable Bitcode XCode 7

### ISSUE THREAD

**Issue by projectAZB** [rollbar/rollbar-ios#23]
I ran into a similar problem that another user had when integrating the Rollbar iOS library into XCode 7 wherein it was telling me to disable bitcode in the app if I wanted the library to work, which I then did, and it still threw the same disable bitcode error. What's the status on this issue?

**Comment by moski** [comment]
any update for this issue, i getting the following error as well:

CrashReporter.framework/CrashReporter(libCrashReporter-iphoneos.a-arm64-master.o)' does not contain bitcode. You must rebuild it with bitcode enabled (Xcode setting ENABLE_BITCODE), obtain an updated library from the vendor, or disable bitcode for this target. for architecture arm64

**Comment by chien** [comment]
Same problem here. Would #26 fix this problem? Could you please review and merge #26 if that works?

**Comment by ezarowny** [comment]
Hey guys, I'll be merging #26 in some form or another really soon. Probably today.

**Comment by ezarowny** [comment]
Can you try using the version that was just merged to master?

**Comment by ezarowny** [comment]
FYI, I'm working to recompile PLCrashReporter so we can use a different prefix than `BIT`.

**Comment by mike-north** [comment]
@ezarowny 
> Can you try using the version that was just merged to master?

Working fine for me in external testing. Thanks!

**Comment by nhnam** [comment]
Awesome works ! Could you guys please update the cocoapods, too? Thanks a lot !

**Comment by ezarowny** [comment]
I'm still working on recompiling PLCrashReporter and testing a sample app. Ideally, we'd use our own prefix so it doesn't collide in the case where someone else is using the `BIT` prefix.

The documentation from Apple isn't super clear regarding bitcode and whether or not compiler flags are required.

**Comment by chien** [comment]
hi @ezarowny,

thanks for the work. I've tried to use the latest master branch with our project and with Cocoapods 1.0.1
The BIT problem is fixed and it successfully reported exceptions.

However, the exception logs are not decoded even after we uploaded dSYM files. Sample log we've seen in our rollbar project is as below. 

Any idea why? or are we missing anything?

```
1
CoreFoundation in 0x182ce8000
2
libobjc.A.dylib in 0x182470000
3
Realm in 0x1016c8000
4
Realm in 0x1016c8000
5
Realm in 0x1016c8000
6
Realm in 0x1016c8000
7
Realm in 0x1016c8000
8
Realm in 0x1016c8000
9
Realm in 0x1016c8000
10
Realm in 0x1016c8000
11
Realm in 0x1016c8000
12
RealmSwift in 0x101d58000
```

**Comment by ezarowny** [comment]
Can you try with the version in #29 ?

**Comment by ezarowny** [comment]
Should be fixed with #29 and 0.2.0
